website/content/partials/deprecation/duplicate-hcl-attributes.mdx

```diff
@@ -0,0 +1,14 @@
+## Duplicate HCL attributes ((#duplicate-hcl-attributes))
+
+| Announced | Expected end of support | Expected removal |
+| :-------: | :---------------------: | :--------------: |
+| JUN 2025  | OCT 2025                | N/A
+
+The usage of duplicate attributes in HCL configuration files and policy definitions is deprecated.
+
+Look for "policy contains duplicate attributes" warnings in the server logs to identify if you are affected, once
+support for duplicate attributes ends any attempts to interact with such policies will result fail, like creating a
+token referencing it or even using a token that references it.
```

Comment:
```suggestion
To find affected policies, look for "policy contains duplicate attributes"
warnings in the server logs. Once we remove support for duplicate attributes,
you cannot interact with affected policies. For example, Vault will return an
error if you try to create a token referencing an affected policy or use an
existing token that references the policy.
```
Style correction: write in active voice, limit 1 idea per sentence, avoid ambiguous pronouns